test_utils/pre_commit/clean_customized_doctypes.py

```diff
@@ -51,15 +51,18 @@ def validate_and_clean_customized_doctypes(customized_doctypes):
 									"value",
 								]:
 									del item[item_key]
-								# if item_key == "modified":
-								# 	item["modified"] = datetime.datetime.now().strftime(
-								# 		DATETIME_FORMAT
-								# 	)
 
 					elif value is None and key not in ["default", "value"]:
 						del file_contents[key]
 
 				if file_contents != original_content:
+					for key, value in list(file_contents.items()):
```

Comment:
This code need to be removed, we do not want to be setting this modified timestamp every time, otherwise we will never be able to commit any code.

Reply:
@agritheory this should update timestamp when the file_content(deleted key value pair if any) and original_content are not same right? do we want to remove this entirely?